release_planning.rb

```diff
@@ -1,8 +1,41 @@
 #!/usr/bin/env ruby
+# frozen_string_literal: true
 
 require 'optparse'
 require_relative 'octokit_utils'
 
+output = File.read('modules.json')
+parsed1 = JSON.parse(output)
+
+class PuppetModule
```

Comment:
This is not used as the objects obtained from json parse of modules.json should contain the necessary data